openadapt/window/_windows.py

```diff
@@ -1,11 +1,63 @@
 from pprint import pprint
+import pygetwindow as gw
 import pickle
 import time
 
 from loguru import logger
 import pywinauto
 
 
+def get_active_window_state_pgw(read_window_data: bool) -> dict:
+    try:
+        active_window = gw.getActiveWindow()
+        if not active_window:
+            raise RuntimeError("No active window found.")
```

Comment:
What do you think about returning `None` instead? I believe this will be handled downstream.